lib/t_web/channels/feed_channel.ex

```diff
@@ -106,7 +107,8 @@ defmodule TWeb.FeedChannel do
     reply =
       %{"mode" => "normal"}
       |> Map.put("since_live_date", since_live_date)
-      |> Map.put("match_expiration_duration", @match_ttl)
+      # TODO remove?
+      |> Map.put("match_expiration_duration", Matches.pre_voicemail_ttl())
```

Comment:
@akhrail1996 is it ok that this now returns two days?

Reply:
@ruslandoga yeah

I thought about releasing backend a bit more - seems that it doesn't make sense to do it prior to new version app release (since it will confuse users too much - they can't exchange voicemail using current app, so effectively for them `match_expiration_duration` will be shortened from 7 to 2 days out of nowhere for new matches)

so its okay to keep it since no need to release backend prior to app release